virtual-desktop-enhancer.ahk

```diff
@@ -354,6 +358,20 @@ OpenDesktopManager() {
     Send #{Tab}
 }
 
+; Let the user change desktop names with a prompt, without having to edit the 'settings.ini'
+; file and reload the program.
+; The changes are temprorary (names will be overwritten by the default values of
+; 'settings.ini' when the program will be restarted.
+ChangeDesktopName() {
+    currentDesktopNumber := _GetCurrentDesktopNumber()
+    currentDesktopName := _GetDesktopName(currentDesktopNumber)
+    InputBox, newDesktopName, Change desktop name, Change the desktop name of desktop #%currentDesktopNumber%, , , , , , , , %currentDesktopName%
```

Comment:
Can you please remove the icon from the popup as well, if possible? It looks odd because it shows the number of the desktop, but since it precedes the title it seems like it should represent the entire application, which it doesn't really. The information about which desktop is going to be changed is also already in the popup body, so the icon is redundant.